ansible.cfg

```diff
@@ -12,3 +12,4 @@ library = ./library
 callback_whitelist = profile_tasks
 roles_path = roles:$VIRTUAL_ENV/usr/local/share/kubespray/roles:$VIRTUAL_ENV/usr/local/share/ansible/roles:/usr/share/kubespray/roles
 deprecation_warnings=False
+inventory_ignore_extensions = ~, .orig, .bak, .ini, .cfg, .retry, .pyc, .pyo, .creds
```

Comment:
Why are you adding .ini and .cfg as ignored? I still use both of these formats.

Reply:
As the param was not previously defined, I just used the default (cf. http://docs.ansible.com/ansible/latest/intro_configuration.html#inventory-ignore-extensions) and added the .creds extension.

Reply:
.ini and .cfg are currently used. Should be removed from the list

Reply:
I don't understand: The default value for inventory_ignore_extensions is a list containing ~, .orig, .bak, .ini, .cfg, .retry, .pyc, .pyo (as indicated in http://docs.ansible.com/ansible/latest/intro_configuration.html#inventory-ignore-extensions), and this is actually the current setting because the parameter is not explicitly set in ansible.cfg.
In order not to introduce a regression, I just took the default list and added .creds to it.
Moreover, as mentioned in http://docs.ansible.com/ansible/latest/intro_configuration.html#inventory-ignore-extensions, this parameter only applies when a directory is used as inventory (like in the referenced issue) and not a single file.

Reply:
Thanks for the explanation.